### PR TITLE
Fix incorrect subject line on supplier emails

### PR DESCRIPTION
**Pull request by richtier**

### CODE REVIEW

File: conf/settings.py [hunk 1]
```diff
@@ -432,3 +432,10 @@
 if env.bool('IP_RESTRICTOR_RESTRICT_UI', False):
     # restrict all pages that are not in apps API, healthcheck, admin, etc
     RESTRICTED_APP_NAMES.append('')
+
+
+# Settings for email to supplier
+CONTACT_SUPPLIER_SUBJECT = env.str(
+    'CONTACT_SUPPLIER_SUBJECT',
+    'Someone is interested in your Find a Buyer profile'
```
Comment (eadpearce) [inline conf/settings.py]: are we still calling it 'Find a Buyer' rather than 'Business Profiles'?

Comment (richtier) [reply conf/settings.py]: the short answer is "yep" - longer answer is

the content is not new - it's copied from https://github.com/uktrade/directory-api/blob/929d4bdd6ce9f7fd64418feffa9de76f3732c8be/conf/settings.py#L367

(we used to send the email via API, now we send via Forms API. When we moved across the subject was not handled correctly)

I'll ask content person and make a follow up change if they want it changed

Comment (kowalcj0) [reply conf/settings.py]: :+1:

File: company/tests/test_views.py [hunk 2]
```diff
@@ -411,7 +412,7 @@ def test_contact_company_view_feature_submit_forms_api_success(
     assert mock_save.call_count == 1
     assert mock_save.call_args == call(
         recipients=['test@example.com'],
-        subject=valid_contact_company_data['subject'],
+        subject=settings.CONTACT_SUPPLIER_SUBJECT,
```
Comment (kowalcj0) [inline company/tests/test_views.py]: :+1: thank you for remembering

**Pull request merged**